wsd/wopi/WopiStorage.cpp

```diff
@@ -779,17 +774,23 @@ void WopiStorage::uploadLocalFileToStorageAsync(const Authorization& auth, LockC
             // Fire the callback to our client (DocBroker, typically).
             asyncUploadCallback(AsyncUpload(AsyncUpload::State::Complete, res));
         };
-
         _uploadHttpSession->setFinishedHandler(std::move(finishedCallback));
 
-        LOG_DBG(wopiLog << " async upload request: " << httpRequest.header().toString());
+        http::Session::ConnectFailCallback connectFailCallback = [asyncUploadCallback]()
+        {
+            // Notify client via callback that the request failed
+            asyncUploadCallback(StorageBase::AsyncUpload(
+                StorageBase::AsyncUpload::State::Error,
+                StorageBase::UploadResult(StorageBase::UploadResult::Result::FAILED)));
```

Comment:
Add message with details: "connection failed"

Reply:
yeah, I was 'just' recapturing the original sequence. But again, missed the Internal Error (exception), duh!